### ISSUE TITLE
Workspace - The new member disappears from the list after being deselected

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#54481]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.77-6
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/5364702&group_by=cases:section_id&group_order=asc&group_id=229065
**Issue reported by:** Applause - Internal Team
**Component** Workspace Settings

## Action Performed:
1.Open the app
2.Navigate to the Workspace editor
3.Click on any Workspace name
4.Click on Members
5.Click on the invite member button
6.Search for an email of an account you have not chatted or interacted before
7.Select member in search result list
8.Unselect above selecting member

## Expected Result:
The new member is still displayed in the list after being deselected

## Actual Result:
The new member disappears from the list after being deselected

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [x] iOS: Standalone
- [x] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/69a3069b-7a98-49e0-b667-db06fc5ef988

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Christinadobrzyn** [comment]
yes I agree that this is expected - Asking QA for a little more information on why this might be a bug. https://expensify.slack.com/archives/C9YU7BX5M/p1735060202932979

**Comment by izarutskaya** [comment]
We have this steps in TestRail and step 8 says: "Verify the default avatar stays the same when selecting and deselecting"

![image](https://github.com/user-attachments/assets/6246247c-4377-4286-82d1-af79232d5404)

**Comment by Christinadobrzyn** [comment]
Ah okay thank you @izarutskaya! 

I think that can be removed as I think it's misleading but I'll double check with the team to confirm this is true.

**Comment by Christinadobrzyn** [comment]
Thanks @parasharrajat - discussion the best method to update the regression test steps here - https://expensify.slack.com/archives/C9YU7BX5M/p1735065262683469?thread_ts=1735060202.932979&cid=C9YU7BX5M

**Comment by Christinadobrzyn** [comment]
Just a heads up that I'm going to be ooo until Dec 30th. I don't think this is urgent and I think it likely just needs an update to the regression test. 

I'm not going to assign this to a BZ teammate but if anything is urgent, please reach out to the team for a volunteer.

**Comment by Christinadobrzyn** [comment]
Following QA chat - https://expensify.slack.com/archives/C9YU7BX5M/p1735868844910009?thread_ts=1735060202.932979&cid=C9YU7BX5M

**Comment by Christinadobrzyn** [comment]
We can close this - QA was able to update the regression tests to account for this check. https://expensify.slack.com/archives/C9YU7BX5M/p1735912637018019?thread_ts=1735060202.932979&cid=C9YU7BX5M
